test/e2e/bootstrapper_test.go

```diff
@@ -75,6 +75,13 @@ func TestBootstrapper(t *testing.T) {
 		assert.True(t, isKubeletRunning(t, kubeletLogPath))
 	})
 
+	t.Run("Update already running kubelet service", func(t *testing.T) {
+		err := wmcb.InitializeKubelet()
+		assert.NoError(t, err, "unable to update kubelet service")
+		time.Sleep(1 * time.Minute)
```

Comment:
Any reason to use 1 minute vs 30 seconds here?

Reply:
Calling InitializeKubelet() on already running service involves stopping the service, updating the config and restarting the service, hence the increased buffer time

Reply:
Not a fan of such sleeps. I would prefer we wait for the expected result with a timeout.